eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlResource.java

```diff
@@ -25,6 +26,7 @@ public EidasSamlResource(EidasAuthnRequestValidator eidasAuthnRequestValidator)
     }
 
     @POST
+    @Valid
```

Comment:
If the @Valid annotation here then don't we need to specify annotations within the ResponseDTO itself so the server knows what to validate?

Reply:
Yep, that's a good point. I'll remove them.

Reply:
That's cool, just making sure my understanding was correct.